spacegrep/src/lib/Semgrep.atd

```diff
@@ -0,0 +1,71 @@
+(*
+   Type definitions for json communication with semgrep.
+
+   Inferred from JSON_report.ml in semgrep-core.
+*)
+
+type match_results = {
```

Comment:
Nice!
I should probably use it in semgrep-core too now that they are in the same repo :)